Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -995,6 +1191,15 @@ BootPickerViewInitialize (
   mBootPicker.Hdr.Obj.OffsetX = mBootPickerView.Width / 2;
   mBootPicker.Hdr.Obj.OffsetY = (mBootPickerView.Height - mBootPicker.Hdr.Obj.Height) / 2;
 
+  // TODO: animations should be tied to UI objects, not global
+  // Each object has its own list of animations.
```

Comment:
Logically yes, but for performance reasons, I think we should stick to a centralised list